intelligence/ts/examples/browser-extension/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "flowerintelligence-firefox-extension",
+  "private": true,
+  "version": "0.0.0",
```

```suggestion
  "version": "0.0.1",
```
Should we maybe use `0.0.1` or is `0.0.0` actually a valid version for most tools?